### PR TITLE
Add version and externalID to API

### PR DESCRIPTION
**Pull request by mattcookio**
Adds the ability (should be backwards compatible once implemented in the service) to filter services by version # and an externalID field

### CODE REVIEW

**Review by TwoSlick — COMMENTED** [review]
Make sure to update the minor version number in setup.py

**Comment by mattcookio** [comment]
> Make sure to update the minor version number in setup.py

Got it @TwoSlick

**Review by TwoSlick — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="serverless-discovery-sdk",
-    version="0.0.5",
+    version="0.1.0",
```
Comment (croutledgeaais) [inline setup.py]: Similar to the JS sdk maybe this should go to 1.0?

**Review by croutledgeaais — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (mattcookio) [reply setup.py]: @croutledgeaais sounds good to me.

**Pull request merged**